extension/content.js

```diff
@@ -372,6 +372,10 @@ document.addEventListener('DOMContentLoaded', () => {
 			if (pageDetect.isPR() || pageDetect.isIssue()) {
 				moveVotes();
 				linkifyIssuesInTitles();
+				addReactionParticipants.add(username);
+				document.addEventListener('click', event => {
+					addReactionParticipants.reapply(event, username);
+				});
```

Comment:
Reactions are also present on comments of commits. E.g. https://github.com/mathiasbynens/dotfiles/commit/fb0bd713f62d1665f4cabed1aa97f7763f18f0b3#commitcomment-16984320